### PR TITLE
docs(ts): fix vtkImageSlice typescript defs

### LINKED ISSUE #2510: ImageSlice vtk.js v24.19.1
use imageSlice has no method named setMapper
when I use react+ts it reports an error like follow Attribute "setMapper" does not exist on type "vtkImageSlice"

### PR DESCRIPTION
**Pull request by daker**
fix #2510

### Changes

- [x] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist

- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing

- [ ] This change adds or fixes unit tests 
- [ ] Tested environment:
  - **vtk.js**: 
  - **OS**: 
  - **Browser**:

### CODE REVIEW

**Comment by floryst** [comment]
LGTM. Thanks, @daker!

File: Sources/Rendering/Core/ImageSlice/index.d.ts [hunk 1]
```diff
@@ -111,6 +117,23 @@ export interface vtkImageSlice extends vtkProp3D {
 	 * and the Renderer will do the images in their own pass.
 	 */
 	hasTranslucentPolygonalGeometry(): boolean;
+
+	/**
+	 * 
+	 */
+	makeProperty(): vtkImageProperty;
```
Comment (floryst) [inline Sources/Rendering/Core/ImageSlice/index.d.ts:124]: `makeProperty()` has the same signature as `vtkImageProperty.newInstance()`

Comment (daker) [reply Sources/Rendering/Core/ImageSlice/index.d.ts:124]: ah, so it needs to take optionally IImageSliceInitialValues right?

File: Sources/Rendering/Core/VolumeProperty/index.d.ts [hunk 2]
```diff
@@ -73,15 +75,15 @@ export interface vtkVolumeProperty extends vtkObject {
...
 	 */
 	getScalarOpacityUnitDistance(index: number): number;
 
 	/**
 	 * Get the currently set gray transfer function. Create one if none set.
-	 * @param {Number} index 
+	 * @param {Number} index
 	 */
-	getGrayTransferFunction(index: number): any;
+	getGrayTransferFunction(index: number): vtkColorTransferFunction;
```
Comment (floryst) [inline Sources/Rendering/Core/VolumeProperty/index.d.ts:86]: This should be a `vtkPiecewiseFunction`.

File: Sources/Rendering/Core/VolumeProperty/index.d.ts [hunk 3]
```diff
@@ -147,37 +149,37 @@ export interface vtkVolumeProperty extends vtkObject {
...
 	setGradientOpacityMinimumValue(index: number, value: number): boolean;
 
 	/**
 	 * Set the color of a volume to a gray transfer function
 	 * @param {Number} index 
-	 * @param func
+	 * @param {vtkColorTransferFunction} func 
 	 */
-	setGrayTransferFunction(index: number, func: any): boolean;
+	setGrayTransferFunction(index: number, func: vtkColorTransferFunction): boolean;
```
Comment (floryst) [inline Sources/Rendering/Core/VolumeProperty/index.d.ts:182]: Ditto here with the `vtkPiecewiseFunction`.

**Pull request merged**